aqueduct/gateway/tests/test_mcp.py

```diff
@@ -0,0 +1,712 @@
+import logging
+from datetime import timedelta
+
+from django.conf import settings
+from pydantic.networks import AnyUrl
+
+from gateway.tests.utils.mcp import MCPLiveServerTestCase
+
+if settings.TESTING:
+    logger = logging.getLogger("aqueduct")
+    logging.disable(logging.NOTSET)
+    logger.setLevel(logging.DEBUG)
```

Comment:
Instead of adding this piece of code to multiple test modules, maybe we could put it somewhere where it would be executed whenever the tests run? Or even in `settings.py`? Then it would also be easier to disable logging altogether, if you only want to run the tests but don't care about the logs.

Reply:
But I'd like to be able to run tests and not have them spit out thousands lines of output that is irrelevant for me, but just show me the errors, if there are any. And now I need to comment out these lines in 2 or 3 files, and then remember not to commit them accidentally. So for me it's a rather inconvenient solution.

By the way, why do MCP tests have to be separated from other tests? What do you mean?